.swiftlint.yml

```diff
@@ -0,0 +1,231 @@
+ - trailing_semicolon # Lines should not have trailing semicolons.
+ - trailing_whitespace # Lines should not have trailing whitespace.
+ - type_body_length # Type bodies should not span too many lines.
+ - type_name # Type name should only contain alphanumeric characters, start with an uppercase character and span between 3 and 40 characters in length.
+ - unneeded_break_in_switch # Avoid using unneeded break statements.
+ - unneeded_parentheses_in_closure_argument # Parentheses are not needed when declaring closure arguments.
+ - untyped_error_in_catch # Catch statements should not declare error variables without type casting.
+ - unused_capture_list # Unused reference in a capture list should be removed.
+ - unused_closure_parameter # Unused parameter in a closure should be replaced with _.
+ - unused_control_flow_label # Unused control flow label should be removed.
+ - unused_enumerated # When the index or the item is not used, .enumerated() can be removed.
+ - unused_optional_binding # Prefer != nil over let _ =
+ - unused_setter_value # Setter value is not used.
+ - valid_ibinspectable # @IBInspectable should be applied to variables only, have its type explicit and be of a supported type
+ - vertical_parameter_alignment # Function parameters should be aligned vertically if they're in multiple lines in a declaration.
+ - vertical_whitespace_closing_braces # Don't include vertical whitespace (empty line) before closing braces.
+ - vertical_whitespace_opening_braces # Don't include vertical whitespace (empty line) after opening braces.
+ - void_return # Prefer -> Void over -> ().
+ - weak_computed_property # Adding weak to a computed property has no effect.
+ - xct_specific_matcher # Prefer specific XCTest matchers over XCTAssertEqual and XCTAssertNotEqual
+ - xctfail_message # An XCTFail call should include a description of the assertion.
+ 
+# This is an entirely separate list of rules that are only run by the analyze command. All analyzer rules are opt-in, so this is the only configurable rule list (there is no disabled/whitelist equivalent).
+analyzer_rules:
+ - unused_private_declaration # Private declarations should be referenced in that file.
+ - unused_import # All imported modules should be required to make the file compile.
+ - explicit_self # Instance variables and functions should be explicitly accessed with 'self.'.
+
+# Configuration for rules
+closure_body_length:
+    warning: 50
+identifier_name: # Identifier names should only contain alphanumeric characters and start with a lowercase character or should only contain capital letters. In an exception to the above, variable names may start with a capital letter when they are declared static and immutable. Variable names should not be too long or too short.
+  excluded:
+    - dt
+    - id
+    - t
+    - p0
+    - p1
+    - p2
+    - p3
+file_length:
+      warning: 1000
+function_body_length:
+    warning: 100
+line_length: 200
```

Comment:
Should we remove it or set our preferred value?

Reply:
As I remember after one more discussion it is up to a developer to decide which line length to use

Reply:
Rule was disabled, but forgot about configuration for `line_length`.
My fault, updated :)